### PR TITLE
[Hub Generated] Review request for Microsoft.ManagementPartner to add version preview/2018-02-01

### PR DESCRIPTION
**Pull request by jetts**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/app/branch/jetts/azure-rest-api-specs/dev-managementpartner-Microsoft.ManagementPartner-2018-02-01...master).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by jetts** [comment]
The changes have been signed off by Anu and Ryan in the private repo: https://github.com/Azure/azure-rest-api-specs-pr/pull/729

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5894 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5894/commits/735b4f0aa1f56f22356f0636741d138258add8e3

File: specification/managementpartner/resource-manager/Microsoft.ManagementPartner/preview/2018-02-01/examples/GetPartnerDetailsNoPartnerId.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "api-version": "2018-02-01"
```
Comment (shahabhijeet) [inline specification/managementpartner/resource-manager/Microsoft.ManagementPartner/preview/2018-02-01/examples/GetPartnerDetailsNoPartnerId.json]: @jetts you updated existing spec with new tags.
Do you mind adding an example that shows the request that has the newly added tags.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Everything else looks good. Once you update the example with tags, this PR is ready to be merged.

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**